FreeRTOS/FreeRTOS
# Add VeriFast kernel queue proofs

Description
-----------
[VeriFast](https://github.com/verifast/verifast) is a deductive verification tool. This PR adds VeriFast proofs for the kernel queue data structure. The proofs give assurance of memory safety, thread safety and function correctness.

Test Steps
-----------
See VeriFast/README.md

Related Issue
-----------
None

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
